packages/smart-contracts/test/contracts/ERC20SingleRequestProxy.test.ts

```
@@ -0,0 +1,293 @@
+  ERC20FeeProxy__factory,
+  UsdtFake,
+  UsdtFake__factory,
+} from '../../src/types';
+import { BigNumber as BN } from 'ethers';
+
+const BASE_DECIMAL = BN.from(10).pow(BN.from(18));
+const USDT_DECIMAL = BN.from(10).pow(BN.from(6));
+
+describe('contract: ERC20SingleRequestProxy', () => {
+  let deployer: Signer;
+  let user1: Signer, user1Addr: string;
```

Biome

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)